lib/configParser.ts

```diff
@@ -0,0 +1,230 @@
+import * as path from 'path';
```

Comment:
it's really confusing that you'll have `configParser.ts` and `configParser.js` in the same directory, seems like the output from the compiler will collide?